server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -59,7 +62,8 @@ case class NettyConfig(
     eventLoopConfig: EventLoopConfig,
     socketConfig: NettySocketConfig,
     initPipeline: NettyConfig => (ChannelPipeline, ChannelHandler) => Unit,
-    gracefulShutdownTimeout: Option[FiniteDuration]
+    gracefulShutdownTimeout: Option[FiniteDuration],
+    serverHeader: Option[String]
```

Comment:
NettyConfig, or NettyOptions? Config is for: "configure the networking layer, the Netty processing pipeline, and start & stop the server.", does it fit?

Reply:
NettyOptions are backend-specific (`NettyCatsServerOptions`, `NettyIdServerOptions`, etc.). `NettyConfig` seemed ok, can't we treat this as "configuring the networking layer"? ;)

Reply:
It's a stretch, but I think we can live with this ;-)